app/cafe.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+from app.errors import (

```

Comment:
It should be a blank line between built-in and your own imports

Reply:
:-)
Fixed.